src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,82 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Cars {
+
+    private static final int MAX_RANDOM_VALUE = 10;
+    private static final String POSITION_SEPARATOR = " : ";
+    private static final String POSITION_MARK = "-";
+
+    private final List<Car> cars;
+
+    public Cars(List<CarName> names) {
+        this.cars = createCars(names);
+    }
+
+    public static Cars fromCars(List<Car> cars) {
+        Cars instance = new Cars(List.of());
+        instance.cars.addAll(cars);
+        return instance;
+    }
```

Comment:
```suggestion
    public Cars(List<CarName> names) {
        this(createCars(names));
    }

    public Cars(List<Cars> cars) {
        this.cars = cars;
    }
```
이와 같이 주 생성자/부 생성자로 구현하면 되지 않을까?